lstchain/calib/camera/calibration_calculator.py

```diff
@@ -141,9 +158,21 @@ def calculate_calibration_coefficients(self, event):
         # Extract calibration coefficients with F-factor method
         # Assume fixed excess noise factor must be known from elsewhere
 
+        numerator = ff_data.charge_std ** 2 - ped_data.charge_std ** 2
+        denominator = self.squared_excess_noise_factor  * (ff_data.charge_median - ped_data.charge_median)
+        gain = np.divide(numerator, denominator, out=np.zeros_like(numerator), where=denominator != 0)
+
+        # correct for the quadratic term if possible
+        if self.quadratic_term is not None:
```

Comment:
as you prefer. B=0 I only suggested because then you can directly use one formula (that would not work with "none"), if you prefer to make it in two steps it can also leave as it is now